go.mod

```diff
@@ -1,19 +1,50 @@
 module github.com/Microsoft/windows-container-networking
 
-go 1.12
+go 1.18
```

Comment:
This is a very big jump in go version. Bumping up go version from 1.12 to 1.18 needs sufficient amount of testing.